proto/p4/p4runtime.proto

```diff
@@ -382,10 +382,28 @@ message Uint128 {
 message SetForwardingPipelineConfigRequest {
   enum Action {
     UNSPECIFIED = 0;
+    // Verify that the target can implement the given config. Do not modify the
```

Comment:
nit: Perhaps s/implement/realize/ since this term seems more frequently used in discussions.